editor.planx.uk/src/pages/FlowEditor/lib/store/settings.ts

```diff
@@ -62,6 +65,21 @@ export const settingsStore: StateCreator<
     return Boolean(result?.id);
   },
 
+  flowCanCreateFromCopy: undefined,
```

Comment:
This is another example (I think) of issues with state management / boilerplate code for a relatively simple feature.

I'd love to dive into this, but it seems like these properties should likely live in either the Apollo cache or route data, as opposed to a global Zustand store.

Again, not advocating that we need a change here now but it's worthwhile revisiting next time we have a few similar settings or fields to manage.